arrows/mvg/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ set( plugin_mvg_headers
   algo/initialize_cameras_landmarks.h
   algo/initialize_cameras_landmarks_basic.h
   algo/triangulate_landmarks.h
-
```

Comment:
Not critical, but I wonder if maybe we should keep these newlines?

Reply:
OK, re-inserted the newlines after algo/...